tests/extractor/extract/test_broadcast_events.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+from unittest import mock
+
+from common.service.errors import BatchCancelled
+from extractor.errors import EmptyResult
+from extractor.service import broadcast_events
+
+
+class BroadCastEventsTestCase(unittest.TestCase):
```

Comment:
The previously existing tests use `pytest` as the test framework. Why use `unittest` ?